intelmq/bots/collectors/http/collector_http.py

```diff
@@ -29,11 +29,25 @@ def process(self):
         self.logger.info("Downloading report from %s" %
                          self.parameters.http_url)
 
-        resp = requests.get(url=self.parameters.http_url, auth=self.auth,
-                            proxies=self.proxy, headers=self.http_header,
-                            verify=self.http_verify_cert,
-                            cert=self.ssl_client_cert,
-                            timeout=self.http_timeout)
+        timeoutretries = 0
+        resp = None
+
+        while timeoutretries < self.http_timeout_max_tries and resp is None:
```

Comment:
Yeah, except that this is specific to timeouts. However, as the error_max_retries can be set to 3 for theses both collectors, I actually think that keeping the number of configuration parameters low is more important than handling the errors differently. So maybe just let the request.get raise the tiumeouterror?